Dockerfile

```diff
@@ -0,0 +1,25 @@
+# Params
+ARG RAILS_ENV="production"
+
+# Defining ruby version
+FROM ruby:2.6.6
+
+# Set working dir and copy app
+WORKDIR /usr/src/app
+COPY . .
+
+# Prerequisites for gems install
+RUN apt-get install tzdata \
+                    git
+
+# Install bundler and gems
+RUN gem install bundler:2.1.4
```

Comment:
I'm not sure we should fix the `bundler` version in the Dockerfile like this. If it needs to be specified (and I'm not sure that it does), I think it should be an arg